develop_cmd.go

```diff
@@ -62,8 +66,28 @@ Please fix your config file in %s and try again.
 			}
 			continue
 		}
+
+		// perform deploy
+		newDeplInfo, err := deployer(config.getProjectConfig().Parse.JSSDK, prevDeplInfo, true, e)
+		if err != nil {
+			if latestError {
+				fmt.Fprintf(
+					e.Err,
+					`Failed to deploy changes to Cloud Code.
+Got error: %s`,
+					errorString(e, err),
+				)
+			} else {
+				latestError = true
+			}
+		} else {
+			printCounter = (printCounter + 1) % printFreq
+			if printCounter == 0 {
```

Comment:
perhaps put 1 here to show first successful deploy instead of 0 which will print printFreq deploys later.